src/rofl/datapath/pipeline/openflow/openflow1x/pipeline/of1x_flow_entry.c

```diff
@@ -19,7 +19,7 @@
 * Intializer and destructor
 */
 
-of1x_flow_entry_t* of1x_init_flow_entry(bool notify_removal){
+of1x_flow_entry_t* of1x_init_flow_entry(bool notify_removal, bool builtin){
```

Comment:
I'd rather suggest to have the default builtin=false and use an additional function to set the flow entry to be builtin. Thus it would be easier to update the other existing code. even notify should be here in first place...